repos/system_upgrade/common/actors/udev/udevadminfo/libraries/udevadminfo.py

```diff
@@ -0,0 +1,19 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import UdevAdmInfoData
+
+
+def process():
+    try:
+        out = run(['udevadm', 'info', '-e'])['stdout']
+    except CalledProcessError as err:
+        raise StopActorExecutionError(
+            message=(
+                "We were unable to gather information about the system devices"
```

Comment:
It's better to stay with passive voice.
```suggestion
                "Unable to gather information about the system devices"
```

Reply:
Done.